### ISSUE TITLE
Support Spigot

### ISSUE THREAD

**Issue by benwebber** [benwebber/ansible-minecraft#7]
- [x] Create a `minecraft_server` variable that defaults to <strike>`vanilla`</strike> `minecraft`
- [x] Download BuildTools
- [x] Run BuildTools with `--rev {{ minecraft_version }}` to build Spigot
- [x] Update Supervisor/systemd templates to run Spigot
- [x] Rename Supervisor/systemd services after implementation
- [x] Update test suite

**Comment by danizen** [comment]
So, should I expect this to work at this point?   Which branch would I use to try it?
Obviously, the JDK has to be installed for this to work...

**Comment by danizen** [comment]
Definitely not working on master branch - I will proceed with the `minecraft_url` setting to download a copy of Spigot I've built and marshalled in place.

**Comment by benwebber** [comment]
`master` worked as of December:

https://travis-ci.org/devops-coop/ansible-minecraft/jobs/315195888

It is possible that the Spigot build script has changed. I don't personally use Spigot. Can you share the error message you got using the `master` branch?

**Comment by jaygooby** [comment]
@danizen If you used ansible-galaxy to install from galaxy (`ansible-galaxy install devops-coop.minecraft`), then you won't have master with spigot support (setting `minecraft_server: spigot` doesn't fail - it just fetches and installs vanilla minecraft).

You need to use the github repo directly:

```
ansible-galaxy install git+https://github.com/devops-coop/ansible-minecraft.git
```

This was still giving me issues, but it's a simple fix: https://github.com/devops-coop/ansible-minecraft/pull/25/commits/ff6c5c8fec0686d445cc7e1e31a24dfb69f5052b

**Comment by danizen** [comment]
Fair enough.  FWIW, we all know we should mark an issue as resolved when it is fixed, and closed when the version containing the fix has "shipped" or been installed.

At the same time, if I use a role for work, I generally clone it and use it from my own repository rather than galaxy, because I don't want it to change under me, and I don't want to manage a galaxy repository in addition to our git installation!
